packages/cli/templates/vue3/src/components/AssetSelector.vue

```diff
@@ -12,16 +12,40 @@ const imageUrl = ref('')
 const handleSelectAsset = async () => {
   imageUrl.value = await props.selectAsset()
 }
+
+const removeAsset = async () => {
+  imageUrl.value = ''
+}
 </script>
 
 <template>
-  <div class="asset-selector">
+  <div>
+    <h2>Asset Selector</h2>
+    <img
+      :src="imageUrl"
+      v-if="imageUrl"
+    />
     <button
-      class="btn"
+      v-if="imageUrl"
+      class="btn w-full"
+      @click="removeAsset"
+    >
+      Remove Asset
+    </button>
+    <button
+      v-else
+      class="btn w-full"
       @click="handleSelectAsset"
     >
       Select Asset
     </button>
-    <span>Image Url: {{ imageUrl }}</span>
   </div>
 </template>
+
+<style scoped>
```

Comment:
lets move all the styles into styles.css that way it will be the same for every framework and we do not need to maintain it in multiple places

Reply:
good idea!